packages/semcom-components/lib/components/sem-com-upload-form.component.ts

```diff
@@ -0,0 +1,198 @@
+import { html, unsafeCSS, css, CSSResult, TemplateResult, property, PropertyValues, state } from 'lit-element';
+import { Theme } from '@digita-ai/ui-transfer-theme';
+import { RxLitElement } from 'rx-lit';
+import { ActorRef, Interpreter } from 'xstate';
+import { FormActors, FormCleanlinessStates, FormEvent, FormRootStates, FormSubmissionStates, FormValidationStates, FormEvents } from '@netwerk-digitaal-erfgoed/solid-crs-components';
+import { from } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { SemComRegisterContext } from './sem-com-register.machine';
+
+export class SemComUploadFormComponent extends RxLitElement {
+
+  /** The actor controlling this component. */
+  @property({ type: Object })
+  // Should not be any - due to ts strict mode (any used to be type Interpreter<SemComRegisterContext>;).
+  public actor?: any;
+
+  /** The actor responsible for form validation in this component.  */
+  @state()
+  // Should not be any - due to ts strict mode (any used to be type ActorRef<FormEvent>;).
+  formActor?: any;
+
+  /** Indicates if if the form validation passed. */
+  @state()
+  isValid? = false;
+
+  /** Indicates if one of the form fields has changed. */
+  @state()
+  isDirty? = false;
+
+  /** Hook called on at every update after connection to the DOM. */
+  async updated(changed: PropertyValues): Promise<void> {
+
+    super.updated(changed);
+
+    if (changed && changed.has('actor') && this.actor) {
+
+      this.subscribe('formActor', from(this.actor).pipe(
+        // Should not be any - due to ts strict mode.
```

Comment:
No, this is simply due to strict mode.

As stated in the comment  on line 19 and line 14 the actual types of the actor and formActor should not be any, but they are because the `from(this.actor)` on line 37, and the `from(this.formActor)` on line 46 and 55 cause errors when they are typed correctly. Because they are any's, if I dont type the machineState as any it won't be able to find machineState.children or machineState.matches. If the  subscribe works correctly with the `from(...)` call, a type is not necessary. It's just a temporary fix.